esp-hal-common/src/spi.rs

```diff
@@ -1330,6 +1330,14 @@ pub trait Instance {
                 .set_bit()
         });
 
+        #[cfg(esp32c6)]
+        unsafe {
+            let pcr = &*esp32c6::PCR::PTR;
+
+            // use default clock source PLL_F80M_CLK
+            pcr.spi2_clkm_conf.modify(|_, w| w.spi2_clkm_sel().bits(1));
```

Comment:
```suggestion
            // use default clock source PLL_F80M_CLK
            (&*esp32c6::PCR::PTR).spi2_clkm_conf.modify(|_, w| w.spi2_clkm_sel().bits(1));
```

No point binding to a variable here if it's only referenced once.